### PR TITLE
Add checking for if warning actually exists in delwarn

### PR DESCRIPTION
**Pull request by emilianya**

### CODE REVIEW

**Comment by emilianya** [comment]
This adds delwarn checking for warning existing before saying it was deleted correctly

File: commands/delwarn.js [hunk 1]
```diff
@@ -27,16 +27,32 @@ module.exports = {
...
 				console.log(err);
-				con.end();
 			} else {
-				message.channel.send(successEmbed(`The warning with the ID of **${warningId}** has been removed.`));
-				console.log("1 warning removed");
-				con.end();
+				console.log(result);
+				if (result.length <= 0) {
+					con.end();
+					return message.channel.send(errorEmbed("There is no warning with this id!"));
```
Comment (nyakase) [inline commands/delwarn.js]:
```suggestion
					return message.channel.send(errorEmbed("There is no warning with this ID!"));
```
For consistency, as IDs are referred to as "ID" and not "id" elsewhere

File: .vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "PreviewInSolutionExplorer": false
+}
```
Comment (nyakase) [inline .vs/VSWorkspaceState.json]: Pretty sure you didn't mean to put the `.vs` folder back in here.

**Review by nyakase — CHANGES_REQUESTED** [review]

**Comment by emilianya** [comment]
I made the requested changes.

File: config.schema.json [hunk 3]
```diff
@@ -67,6 +67,16 @@
...
+                },
+                "warn": {
+                    "description": "Options for the warn command",
+                    "type": "object",
+                    "properties": {
+                        "mysql": {
+                            "description": "Toggles if a MySQL database should be used",
+                            "type": "boolean"
+                        }
+                    }
```
Comment (nyakase) [inline config.schema.json]: I just noticed - what's up with this? Warn doesn't its own config (`config.commands.warn` doesn't exist) and `config.misc.mysql` has already been implemented

Comment (emilianya) [reply config.schema.json]: I am not sure, I think it might be a relic from the past when warn existed as the only mysql function.

Comment (nyakase) [reply config.schema.json]: You can remove it, as having it there would present `config.commands.warn` as something you can create even though it doesn't do anything

Comment (viluzi) [reply config.schema.json:79]: I have class starting so this might never happen lol

**Review by nyakase — APPROVED** [review]

**Pull request merged**